source/_patterns/02-organisms/content-headers/content-header-journal.mustache

```diff
@@ -89,33 +89,33 @@
       </div>
     </div>
     {{^isOneColumn}}
-      <div class="content-header-grid__side">
+      <div class="grid-side">
```

Comment:
Can we name this as side-section

Reply:
Thanks, "side-section" sounds better, I rename all classes to start like this